site/docs/components/card/examples.mdx

```diff
@@ -10,46 +10,57 @@ data:
 
 <LivePreviewControls>
 
-    <LivePreview componentName="card" exampleName="Default" >
-      ### Default
-    </LivePreview>
+    <LivePreview componentName="card" exampleName="Default" displayName="Card" >
```

Comment:
I think I mentioned this on LinkCard but there's a bug when this is indented where the heads don't appear in the TOC